src/components/menu.html

```diff
@@ -1 +1,23 @@
+<div class="wrap_menu">
 
+    <div class="wrap_trademark">
+            <p class="trademark">HOFMANN</p>
+    </div>
+
+<div class="wrap_list1_menu">
+    <ul class="list1_menu">
+        <li class="list1">HOTELS</li>
```

Comment:
i think u should move to component and use list tag to list item
example: [list](https://github.com/baonguyen26/html-css/blob/main/src/components/list.html)

Reply:
i will fix that on next pull request